ezbake/config/conf.d/auth.conf

```diff
@@ -10,7 +10,30 @@ authorization: {
             }
             allow: "$1"
             sort-order: 500
-            name: "puppetlabs catalog"
+            name: "puppetlabs v3 catalog from agents"
+        },
+        {
+            # Allow nodes to retrieve their own catalog
+            match-request: {
+                path: "^/puppet/v4/catalog/([^/]+)$"
+                type: regex
+                method: [get, post]
+            }
+            # allow: "$1"
+            deny: "*"
```

Comment:
Are we including this version now to support the work @tvpartytonight is doing? I don't think we have plans to officially support these requests any time soon, right?

Reply:
No, I just thought we'd break out the possible workflows of what's possible. Fine to remove what we aren't publicly supporting yet.

Reply:
Updated. I think I've addressed this, lemme know if not.